src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String string;
+    private int variable;
+    private int age;
+    private int size;
+
+    public static void main(String[] args) {
+        Dog dog = new Dog();
+        Cat cat = new Cat();
+    }
+
+    private String getFromOneToTen() {
+        String result = "";
+
+        for (int i = 0; i < 10; i++) {
+            if (i % 2 == 0) {
+                result = result + i + " ";
+            }
+        }
+        return result;
+    }
+
+    private void sayHello(String hello) {
+        System.out.println("Hello, Mate! I am happy to see you! This class looks weird!"
+                +
```

Comment:
The same with `+` operator at the beginning of the line, I would kindly ask to add this operator at the begginning of each line and the rest of the text after that